conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -48,8 +48,10 @@ def __init__(self, conanfile, namespace=None):
         self._target = None
 
         self.apple_arch_flag = self.apple_isysroot_flag = None
-
         self.apple_min_version_flag = apple_min_version_flag(self._conanfile)
+
+        self.msvc_runtime_flag = self._get_msvc_runtime_flag()
```

Comment:
But this flag is not used at all by the toolchain?

Reply:
Wow, I don't know what happened with the rest of the code 😅 . It is fixed now